code/common/finspace.q

```diff
@@ -0,0 +1,6 @@
+// Config for setting Finspace specific parameters
+\d .finspace
+
+enabled:@[value;`enabled;0b];                               //whether the application is finspace or on prem - set to false by default
+database:@[value;`database;"database"];                     //name of the finspace database applicable to a certain RDB cluster - Not used if on prem
+clusters:@[value;`clusters;enlist `cluster];                //list of clusters to be reloaded during the rdb end of day (and possibly other uses)
```

Comment:
this should maybe be called `hdbclusters` or something? if I'm interpreting the comment correctly it'll just be a list of HDB clusters that need to be reloaded, right?

Reply:
I'm currently using this flag in my feedhandler for creating a connection string to the rdb cluster. I can create a flag a seperate flag for that I suppose. I might just create that within the feedhandler itself as it's not something that will need to be used once a TP is available. I will remain this flag to hdbclusters as it will come in handy for HDB reloads